lib/Board.rb

```diff
@@ -0,0 +1,95 @@
+class Board
+    def initialize
+        @board = Array.new(3){Array.new(3)}
+        #@board = [[3,8,1], [3,2,1], [1,1,1]]
+    end
+   # cordinate = { "1" => [0,0], "2" => [0,1], "3" =>[0,2], "4" => [1,0], "5" => [1,1], "6" =>[1,2],"7" => [2,0], "8" => [2,1], "9" =>[2,2] }
+    def add_piece(position, piece)
+        if is_position_available(position)
+            @board[position[0]][position[1]] = piece
+            true
+        else 
+            puts "Posision is not available try another.."
+            false
+        end
+    end
+
+    def is_position_available(pos) 
+        @board[pos[0]][pos[1]].nil?
+    end
+
+   def show_board()
```

Comment:
This method mixes your Board logic with user interface. Probably it would be better for you to place this in a UI-related module?